users/bocaj/process_records.c

```diff
@@ -0,0 +1,141 @@
+#include "bocaj.h"
+#include QMK_KEYBOARD_H
+
+uint16_t copy_paste_timer;
+uint16_t grave_layer_timer;
+
+__attribute__ ((weak))
+bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
+  return true;
+}
+
+// Defines actions tor my global custom keycodes. Defined in bocaj.h file
+// Then runs the _keymap's record handler if not processed here
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case KC_MWRK:
+      if (!record->event.pressed) {
+        set_single_persistent_default_layer(_WORKMAN);
+        layer_move(0);
+        ergodox_blink_all_leds();
+      }
+      break;
+    case KC_WWRK:
+      if (!record->event.pressed) {
+        set_single_persistent_default_layer(_WINWORKMAN);
+        layer_move(0);
+        ergodox_blink_all_leds();
+      }
+      break;
+    case KC_MQWR:
+      if (!record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY);
+        layer_move(0);
+        ergodox_blink_all_leds();
+      }
+      break;
+    case MC_LOCK:
+      if (!record->event.pressed) {
+        layer_move(0);
+        SEND_STRING(SS_LCTRL(SS_LGUI("q")));
+      }
+      break;
+    case KC_MAKE:  // Compiles the firmware, and adds the flash command based on keyboard bootloader
+      if (!record->event.pressed) {
+        uint8_t temp_mod = get_mods();
+        clear_mods();
+        send_string_with_delay_P(PSTR("make " QMK_KEYBOARD ":" QMK_KEYMAP), 10);
+        if (temp_mod & MODS_SHIFT_MASK) {
+          #if defined(__ARM__)
```

Comment:
This should be lower case. 

```suggestion
           #if defined(__arm__)
```

Otherwise, it's not detected properly.